src/encoded/schemas/sc_atac_multiplet_quality_metric.json

```diff
@@ -128,6 +131,7 @@
         },
         "barcode_pairs_multiplets": {
             "title": "Multiplet barcode pair statistics",
+            "description": "Statistics for each barcode pair in a multiplet. The 'PrimaryBarcode' field specifies the highest-count barcode of the multiplet ",
```

Comment:
Can we remove the extra space at the end of this line?

Reply:
done